### ISSUE TITLE
BUG create new virtual server (subdomain) proxy to LAN IP, Failed to apply changes :  AH00526: SuexecUserGroup

### ISSUE THREAD

**Issue by chris001** [virtualmin/virtualmin-gpl#442]
```
Failed to apply changes :

AH00526: Syntax error on line 2 of /etc/apache2/sites-enabled/proxied.sub.mydomain.tld.conf:
Invalid command 'SuexecUserGroup', perhaps misspelled or defined by a module not included in the server configuration
Action 'configtest' failed.
The Apache error log may have more information.
```

`Configtest` works when you comment out the 2 lines of "SuexecUserGroup".
PHP mode was originally "None", then set to "FPM" (the only choices).  Regardless of PHP mode, while the two "SuexecUserGroup" lines were active, `configtest` always failed.

Probably should only uncomment `SuexecUserGroup` lines when PHP mode is `Suexec` ?

**Comment by jcameron** [comment]
Looks like Virtualmin thought that `suexec` support for running CGI scripts is available when really it isn't.

What does it say about how CGI scripts are run if you SSH in and run `virtualmin check-config` ?

**Comment by chris001** [comment]
```
sudo virtualmin check-config
Your system has 3.83 GiB of memory, which is at or above the Virtualmin recommended minimum of 256 MiB

BIND DNS server is installed

Mail server Postfix is installed and configured

Postfix can support per-domain outgoing IP addresses, but is not currently configured to do so. This can be setup in the Postfix Mailserver module

Apache is installed

Apache supports HTTP/2 on your system

The following PHP execution modes are available : fpm

The following PHP-FPM versions are available : 8.1.2 (php8.1-fpm)

Apache is configured to host SSL websites

MariaDB 10.6.7 is installed and running

Logrotate is installed

SpamAssassin and Procmail are installed and configured for use

ClamAV is installed and assumed to be running

Using network interface eth0 for virtual IPs

IPv6 addresses are available, using interface eth0

Default IPv4 address for virtual servers is x.x.x.x

Virtualmin could not work out the default IPv6 address for virtual servers on your system. You will need to update either the Network interface for virtual addresses or Default virtual server IP address fields on the module configuration page.

Quotas are not enabled on the filesystem / which contains home directories under /home and email files under /var/mail. Quota editing has been disabled.

Shell /bin/false for FTP users is not included in /etc/shells, which may prevent FTP access

All commands needed to create and restore backups are installed

The selected package management and update systems are installed

Chroot jails are available

OK
```

**Comment by jcameron** [comment]
Ok I see the bug that causes this - it will be fixed in the next Virtualmin release.
